src/components/Episodes/context.jsx

```diff
@@ -1,23 +1,29 @@
-import React, { createContext } from "react";
+import React, { createContext, useState } from "react";
 import { data } from "./";
 
 const EpisodesContext = createContext();
 
-const episodes = data.length;
-
-const initialState = {
-	activeIndex: 0,
-	episodeCount: episodes
-};
 
 function EpisodesProvider(props){
 	const { children } = props;
 
+	const episodeCount = data.length;
+	const [activeIndex, setActiveIndex] = useState(0);
+	const [overlayActive, setOverlayActive] = useState(false);
```

Comment:
Let's restore the `initialState` object and use the values from that as the default values for these two `useState` calls